docs/concepts.rst

```diff
@@ -70,7 +70,9 @@ to several objects.
 Authentication
 ==============
 
-Kinto doesn't provide users management, like user sign-up, password modification, etc.
+Kinto comes with an accounts API that allows user sign-up, password modification and user deletion. It allows you to configure an administrator account that has read/write access to all other accounts.
+
+By default, Kinto accounts are authenticated using basic authentication.
```

Comment:
Here I would mention that Kinto is not coupled to any authentication mecanism or provider. And then in another paragraph mention there's an `account` built-in plugin that provide user sign-up, ... etc with a link to the docs ``` :ref:`link <api-accounts>` ```